<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>removerPalavra</class>
 <widget class="QMainWindow" name="removerPalavra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>60</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>REMOVER</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>favicon.ico</normaloff>favicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(154, 154, 154);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="inputID">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
border-radius: 8px;
padding: 2px;
color: #000;
background-color: rgb(200, 200, 200); 
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnApagar">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: 2px solid;
	border-radius: 8px;
	color: #878787;
	background-color: rgb(200, 200, 200); 
}

QPushButton:hover {
	color: rgb(200, 200, 200);
	background-color: #878787;
}

QPushButton:pressed {
	color: rgb(200, 200, 200);
	background-color: black;
}</string>
      </property>
      <property name="text">
       <string>APAGAR</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icones.qrc"/>
 </resources>
 <connections/>
</ui>
